tests/sentry/api/endpoints/test_relay_globalconfig_v3.py

```diff
@@ -78,3 +78,26 @@ def test_return_global_config_on_right_version(
         assert "global" not in result
     else:
         assert result.get("global") == {"global_mock_config": True}
+
+
+@patch(
+    "sentry.relay.globalconfig.get_global_generic_filters",
+    lambda *args, **kwargs: {
+        "version": 1,
+        "filters": [
+            {
+                "id": "test-id",
+                "isEnabled": True,
+                "condition": {
+                    "op": "eq",
+                    "name": "event.contexts.browser.name",
+                    "value": "Firefox",
+                },
+            }
+        ],
+    },
+)
+@patch("sentry.relay.globalconfig.RELAY_OPTIONS", [])
+def test_global_config_valid_with_generic_filters():
+    config = get_global_config()
+    assert config == normalize_global_config(config)
```

Comment:
Nice! I would add a test like this for project configs as well, to be safe.

Reply:
lol, I must have discarded it dealing with git. Should be back in [5031c20](https://github.com/getsentry/sentry/pull/66410/commits/5031c2003dd80040e3cc4c9dda440b2ff6471ee7).